bin/install

```diff
@@ -58,13 +58,20 @@ install_elixir_ref() {
   # Expand source archive
   tar zxf $source_path -C $install_path --strip-components=1 || exit 1
 
-  # Build from source in a subshell
-  # we don't want to disturb current working dir
-  (cd $install_path; make clean test)
+  # Build from source in a subshell because we don't want to disturb current working dir
+  # installing using DESTDIR makes sure there are no tempfiles in the install dir
+  # and fixes compatibility issues with the elixir-ls language server project
+  (cd $install_path; make clean test DESTDIR=$install_path install)
```

Comment:
Because this `make` command is run in `$install_path` are you sure this fixes the issue? All the files are already in this directory because they are placed here by `tar` on line 59.

Reply:
Hi :) At first I implemented it by separating the "untar"-dir and the "build"-dir, and then moving the completed build. This works fine for creating a "working" elixir, but broke the ability of VS code to open the source code of elixir itself (which is basically the reason for installing elixir from source in the first place, afaik). I didn't really understand why VS Code couldn't read those source files, as far as I could tell they were right there but maybe the directory structure was slightly different. Also I ran out of time to experiment further with it :p

It seems this works because the `install` command of the makefile does some cleaning before it starts the build, but I'm not sure: https://github.com/elixir-lang/elixir/blob/master/Makefile#L117

By doing it like this, vscode-click-to-sources ability works and elixir-ls works as expected as well. Although I agree that it isn't very clean like this, and fragile to changes of the upstream Elixir makefile. 

Anyway, it would probably be a good idea if somebody else tested it as well before it is merged. Did you try it yourself?

Reply:
I did test this myself and it seemed to work fine. I wasn't able to replicate the issue this was fixing with the original code, but I don't see any reason not to merge it if the `install` target cleans things up in the install directory before installing.